tests/Microsoft.Fx.Portability.MetadataReader.Tests/ReflectionMetadataInfoTests.cs

```diff
@@ -46,7 +46,7 @@ public void UnresolvedAssemblyTest()
             Assert.Equal(s_expectedResult.Count(), actual.Count());
 
             // Use this instead of Assert.Equal so it will output the missing item
-            foreach (var items in s_expectedResult.Zip(actual, Tuple.Create))
+            foreach (var items in actual.Zip(s_expectedResult, Tuple.Create))
```

Comment:
Why the change? I think that [Assert.Equal](https://github.com/xunit/xunit/blob/master/src/xunit.assert/Asserts/EqualityAsserts.cs) expects the 'expected' value first and I think [Zip](https://msdn.microsoft.com/library/dd267698%28v=vs.100%29.aspx) will take items from the calling collection as the 'first' ones, right?